docs/Wallet-System.md

```diff
@@ -16,7 +14,14 @@ theoretically be able to scale to hundreds of thousands of
 wallets/accounts/addresses.
 
 Each account can be of a different type. You could have a pubkeyhash account,
-as well as a multisig account, a witness pubkeyhash account, etc.
+as well as a multisig account, a witness pubkeyhash account, etc. This is where
+the bcoin wallet begins to deviate from strict BIP44: Accounts in a bcoin wallet
```

Comment:
There can also be accounts _after_  other accounts without any transactions, which also deviates and creates issues with account discovery (https://github.com/bcoin-org/bcoin/issues/698).

Reply:
This also emphasizes the need to backup the wallet database.